lms/static/js_test.yml

```diff
@@ -64,10 +65,12 @@ lib_paths:
 src_paths:
     - js
     - js/common_helpers
+    - teams/js
```

Comment:
We should create a wiki page that enumerates all the steps to add that you went through to add a new Django app to the platform.